index.html

```diff
@@ -1100,6 +1070,12 @@
   </p>
   </section> <!-- NFCRecordType interface -->
 
+  <section><h3>The <strong>NFCRecordData</strong> type</h3>
+    <pre class="idl">
+      typedef (DOMString or unrestricted double or object or ArrayBuffer) NFCRecordData;
```

Comment:
Totally optional, but you can just embed this typedef in the same IDL block as `NFCRecord`. It doesn't really need its own section.